tests/selectedtests/work_items/test_task_mappings_work_item.py

```diff
@@ -27,46 +27,41 @@ def test_create_new_task_mappings(self):
         assert not work_item.start_time
         assert not work_item.end_time
 
-    def test_no_module_passed_in(self):
-        collection = MagicMock()
-        collection.insert_one.return_value.acknowledged = True
-
+    def test_no_module_passed_in(self, mock_task_mappings_queue):
+        mock_task_mappings_queue.insert_one.return_value.acknowledged = True
         work_item = under_test.ProjectTaskMappingWorkItem.new_task_mappings(
             PROJECT, SOURCE_FILE_REGEX
         )
-        new_item = work_item.insert(collection)
+        new_item = work_item.insert()
 
-        collection.insert_one.assert_called_once()
+        mock_task_mappings_queue.insert_one.assert_called_once()
 
-        assert new_item
-
-    def test_insert_adds_item_to_collection(self):
-        collection = MagicMock()
-        collection.insert_one.return_value.acknowledged = True
+        assert new_item is True
```

Comment:
A little refactor, I think we should check that `True` is returned, rather than just a truthy value.